sympy/calculus/util.py

```diff
@@ -789,10 +789,13 @@ def maximum(f, symbol, domain=S.Reals):
     1/2
 
     """
-    if isinstance(domain, EmptySet):
-        return S.EmptySet
+    if type(symbol) == Symbol:
```

Comment:
we use `isinstance`. This does not check subtypes.

Reply:
I used `isinstance` in commit https://github.com/sympy/sympy/pull/16473/commits/91410ce09db91a2a17b2fe24e3cc105de68b58b2 . Locally, the tests are running fine with this change but Travis tests are failing beacuse of this. Maybe I need to import `Symbol` gain while defining `maximum`. Let me check

Reply:
Yes, it is working. Hence, I mark this conversation resolved.